README.md

```diff
@@ -37,23 +38,33 @@ The market lacks a tool for developers to compare different database products be
 
 This project is under development and is very unstable. The way to start this project may improve as process goes on.
 
-### Installing Frontend Dependencies
-```
-cd ./frontend && pnpm i
-```
-
 ### Fetching Data
+Sample data is provided at [here](https://github.com/bytebase/dbcost/blob/main/store/data/sample.json). Your may start dbcost in sample mode, without fetching your own data. If you decide to start with sample data, just skip this step.
+
+Or, if you would like to fetch the newest data online, please apply for a [GCP API KEY](https://cloud.google.com/apigee/docs/api-platform/security/api-keys?hl=en&_ga=2.248673159.1879775959.1652425553-1923221278.1651636210&_gac=1.19919946.1652425553.Cj0KCQjwg_iTBhDrARIsAD3Ib5jxpCArIqLqlXF61hULy8r46r2GwzWKde-4nLdzetVquoP6ed1R_QsaAjXVEALw_wcB) with access to the [Cloud Billing API](https://cloud.google.com/billing/docs/reference/rest) yourself first. 
```

Comment:
```suggestion
Or, if you would like to fetch the newest data online, please apply for a [GCP API KEY](https://cloud.google.com/apigee/docs/api-platform/security/api-keys) with access to the [Cloud Billing API](https://cloud.google.com/billing/docs/reference/rest) yourself first. 
```

Reply:
Please list instructions for each cloud provider respectively. If AWS doesn't need the key, we also mention it explicitly.